abdiff/core/calc_ab_metrics.py

```diff
@@ -238,12 +239,12 @@ def _get_source_counts(
     sources: list[str],
     analysis: dict,
 ) -> dict:
-    any_field_modified_condition = " OR ".join(f"{field} = 1" for field in fields)
```

Comment:
This is indicative of the key change from the PR.  We no longer need to dynamically construct a SQL query to look for a True `1` value for _any_ TIMDEX field, field names we do not know in advance.

Instead, we can just apply the simple, static clause `where has_diff` in its place anywhere we want to identify records with diffs.